plugins/destination/file/Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+.PHONY: lint
+lint:
+	@golangci-lint run --config ../../.golangci.yml --timeout 10m --verbose
```

Comment:
We can remove the timeout and verbose flags (we have the timeout in the config file, and verbose generates too much information for it to be useful)

Reply:
> Looks good, still need to test it locally. Also since we have only 2 file types maybe do it as a package inside the plugin, not sure, to avoid the monorepo complexities, and only move it outside when it's used somewhere else than this plugin.
> 
> For example we don't have CI that runs the tests/linting/etc. on this module, so we'll need to create a separate one (or do it inside the file plugin CI)

Im going to open a PR for s3, gcs and azure in a moment after this one is merged so we need it to be a package